tests/TestKernel.php

```diff
@@ -85,7 +123,7 @@ function (ContainerBuilder $container) {
     public function loadRoutes(LoaderInterface $loader): RouteCollection
     {
         $routes = new RouteCollectionBuilder($loader);
-        $routes->import(__DIR__.'/../Resources/config/routing.xml');
+        $routes->import(__DIR__.'/../src/Resources/config/routing.xml');
```

Comment:
If you use MicroKernelTrait, you can register the routes in the kernel without the external XML file - keeps things simple.